include/SFML/Audio/SoundStream.hpp

```diff
@@ -281,7 +299,7 @@ class SFML_AUDIO_API SoundStream : public SoundSource
 
     enum
     {
-        BufferCount = 3 ///< Number of audio buffers used by the streaming loop
+        BufferCount = 3,    ///< Number of audio buffers used by the streaming loop
```

Comment:
What purpose does this modification serve, apart from making the code incorrect C++98?